packages/php/email-editor/src/Engine/class-email-editor.php

```diff
@@ -184,15 +184,18 @@ private function get_default_email_post_args(): array {
 	 * @return void
 	 */
 	private function register_email_post_send_status(): void {
+		$default_args = array(
+			'public'                    => false,
+			'exclude_from_search'       => true,
+			'internal'                  => true, // for now, we hide it, if we use the status in the listings we may flip this and following values.
+			'show_in_admin_all_list'    => false,
+			'show_in_admin_status_list' => false,
+			'private'                   => true, // required by the preview in new tab feature for sent post (newsletter). Posts are only visible to site admins and editors.
+		);
+		$args         = apply_filters( 'mailpoet_email_editor_post_send_status_args', $default_args );
```

Comment:
The status is `sent` so perhaps the better name for the hook would be `mailpoet_email_editor_post_sent_status_args`. What do you think? We could probably also change the method name.

Reply:
I agree with you. Let's do that.